frontend/src/App.js

```diff
@@ -141,6 +142,12 @@ const App = () => {
               <Route path="/history/:series/:builds">
                 <History />
               </Route>
+              <Route exact path="/team">
+                <Team />
+              </Route>
+              <Route path="/team/:name">
```

Comment:
Team names will most likely stay as strings with no special characters so this should be okey.